com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/handler/VariablesRequestHandler.java

```diff
@@ -94,6 +95,24 @@
         }
 
         VariableProxy containerNode = (VariableProxy) container;
+
+        if (containerNode.getReferencedVariableId() != null && DebugSettings.getCurrent().showToString) {
```

Comment:
I'd suggest to extract the following block to a private method like `resolveLazyValue(variable)` for more readable.

Reply:
fixed in [c8366d5](https://github.com/microsoft/java-debug/pull/401/commits/c8366d51f901ca3fae535aadda0f4c75b2354bb7)